src/server/environment.js

```diff
@@ -1,27 +1,17 @@
 
-const { tmpdir } = require('os')
-const { mkdtempSync } = require('fs')
 const { join, resolve } = require('path')
 
-function tempDataDir () {
-  return mkdtempSync(join(tmpdir(), 'arpa-data-'))
-}
-
 const VERBOSE = Boolean(process.env.VERBOSE)
 
-const DATA_DIR = resolve(process.env.DATA_DIR || tempDataDir())
-const UPLOAD_DIR = join(DATA_DIR, 'uploads')
-const ARPA_REPORTS_DIR = join(DATA_DIR, 'arpa_reports')
```

Comment:
Note that I am removing the only reference to ARPA_REPORTS_DIR here:

https://github.com/usdigitalresponse/arpa-reporter/pull/243/commits/765ce991dfe2e4e1116c12e118dfb528b1e0a24a#diff-b3759fc859de68384f315c38753e3036f1d4a875d873a490cab83e5a4cc091d6L152